example/app/data/DemoDataSource/plugins/table/car_list/blueprints/Car.blueprint.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Car",
+  "type": "CORE:Blueprint",
+  "description": "",
+  "attributes": [
+    {
+      "name": "type",
+      "attributeType": "string",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
+      "optional": false
+    },
+    {
+      "name": "Manufacturer",
```

the convention we have used elsewhere is that attributes have camelCase and start with a lowercase letter ) 
```suggestion
      "name": "manufacturer",
```